feature/interface/singleton/ate_tests/singleton_test/README.md

```diff
@@ -0,0 +1,92 @@
+# RT-5.1: Singleton Interface
+
+## Summary
+
+Singleton L3 interface (non-LAG) is supported on DUT.
+
+## Procedure
```

Comment:
I updated the test plan. Please use the updated version here.

Reply:
Done!